superset/views/base.py

```diff
@@ -30,6 +30,7 @@
 from flask_appbuilder.models.sqla.filters import BaseFilter
 from flask_appbuilder.security.sqla.models import Role, User
 from flask_appbuilder.widgets import ListWidget
+from flask_jwt_extended.exceptions import NoAuthorizationError
```

Comment:
hmm, I think this package is optional, it might inherit from a generic flask auth exception class

Reply:
😭 looks like it's just inheriting from `Exception`